jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/file/ErrorManageUtil.java

```diff
@@ -157,7 +157,7 @@ public static void omitOpsAlert() {
             Map<String, List<String>> assetPermissionMapping = new HashMap<>();
             for (ErrorVH errorVH : entry.getValue()) {
                 List<String> permissionIssues = assetPermissionMapping.get(errorVH.getType());
-                if (errorVH.getType().equals("phd") && (errorVH.getException().contains("AccessDeniedException") || errorVH.getException().contains("SubscriptionRequiredException"))) {
+                if ((errorVH.getType().equals("phd") && (errorVH.getException().contains("AccessDeniedException") || errorVH.getException().contains("SubscriptionRequiredException")))||(errorVH.getType().equals("all") && errorVH.getException().contains("AWSSecurityTokenService"))) {
```

Comment:
@AishwaryaKulkarni97 please properly format spacings in the expression.

Reply:
Addressed in commit